### ISSUE TITLE
Small bug in the Patcher for \$prefix\Twig\Node\ModuleNode

### ISSUE THREAD

**Issue by light-source** [OnTheGoSystems/twig-scoper#5]
Hello OnTheGoSystems team!

Thank you so much for this repository; you saved me tons of time!

I found one small bug:

The patcher for `\$prefix\Twig\Node\ModuleNode` uses `getFormattedPrefix( 1, false )`, but must use 2 as the first argument. Otherwise, it causes issues with specific names. For example, if 'org\some\vendor' is used, '\v' escapes incorrectly.
 
By the way, Twig v3 has changed slightly, so it requires additional fixes. I've created a [scoper config for Twig v3](https://gitlab.com/lightsource/twig-scoper), just in case somebody is interested.

**Comment by zaantar** [comment]
@light-source thank you for your kind words! I'm really happy someone finds this repo useful.

Did you mean this? https://github.com/OnTheGoSystems/twig-scoper/pull/6/files

As for Twig v3, thanks for letting us know. We are not actively using it at this point but I will happily accept PRs for this project if you want to include anything here.

**Comment by light-source** [comment]
@zaantar Yeah, I was integrating Twig into [our plugin](https://wordpress.org/plugins/acf-views/) and didn't want to include it directly, like Timber and other plugins. It's a pity that WP still has no dependency manager. 

Your repository has been a real time-saver! Thanks once again for making it available.

 About the fix - yes, I meant that.
 
Thanks for being open. I already added the fix to our Scoper's config, but if we need it next time, I'll keep in mind that there is an option to improve the current one.
 
 Good luck with your projects!
